include/nuttx/tls.h

```diff
@@ -267,6 +270,13 @@ uintptr_t tls_get_value(int tlsindex);
 int tls_set_value(int tlsindex, uintptr_t tlsvalue);
 #endif
 
+#if CONFIG_TLS_TASK_NELEM > 0
+int task_tls_alloc(tls_dtor_t dtor);
+void task_tls_destruct(void);
+int task_tls_set_value(int tlsindex, uintptr_t tlsvalue);
+uintptr_t task_tls_get_value(int tlsindex);
```

Comment:
Do we need to add comment with AP description? Similar to APIs above

Reply:
Thanks, I'll add it later